packages/user-operation-controller/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [18.0.0]
+
+### Changed
+
+- **BREAKING:** Bump peer depepdency `@metamask/keyring-controller` from `^17.0.0` to `^18.0.0` ([#4915](https://github.com/MetaMask/core/pull/4915))
+- Bump `@metamask/polling-controller` from `^12.0.0` to `^12.0.1` ([#4870](https://github.com/MetaMask/core/pull/4870))
+- Bump `@metamask/transaction-controller` from `^38.0.0` to `^38.3.0` ([#4864](https://github.com/MetaMask/core/pull/4864), [#4882](https://github.com/MetaMask/core/pull/4882), [#4900](https://github.com/MetaMask/core/pull/4900))
```

Comment:
Can probably drop this as it's a change to dev dependencies so it should not affect bundle size.

``` suggestion
```